.github/workflows/ci-frontend-e2e.yml

```diff
@@ -50,15 +53,75 @@ jobs:
           ../api/bin/wait-for-api.sh
         shell: bash
 
-      - name: Run E2E Tests
+      - name: Build a prod version of the site
         run: |
-          npm run build
           cat .env.development >> .env.local
-          npm run test:e2e
+          npm run build
+
+      - name: Run e2e tests (Shard ${{ matrix.shard }}/${{ matrix.total_shards }})
+        env:
+          CI: true
+          TOTAL_SHARDS: ${{ matrix.total_shards }}
```

Comment:
curious about the decision to use environment variables, which I can't find a reference to in the docs, versus the `--shards` flag that seems to be recommended https://playwright.dev/docs/test-sharding#sharding-tests-between-multiple-machines